pages/index.js

```diff
@@ -1,18 +1,52 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+
 import Head from "next/head";
-import styled from "styled-components";
+import ShoppingList from "../components/ShoppingList";
 
 export default function Home() {
+  const [shoppingListItems, setShoppingListItems] = useState(shoppingListDB);
+
   return (
     <div>
```

Comment:
You can use a fragment element here, since you don't really need the extra hierarchy: `<> ... </>`